spec/controllers/retention_schedules_controller_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
+RSpec.describe RetentionSchedulesController, type: :controller do
+  let(:team_admin) { find_or_create :team_admin }
+
+  let(:case_with_rrd) {
+    create(
+      :offender_sar_case, :closed, :with_retention_schedule,
+      planned_destruction_date: Date.tomorrow
+    )
+  }
+
+  let(:retention_schedule) { case_with_rrd.retention_schedule }
+
+  # TODO: implement proper access control and add tests here when ready.
```

Comment:
👍 - probably easiest to follow the @user.team_admin? approach that I did in my most recent PR, but you've probably already thought about this :-)